doc/sphinx/addendum/implicit-coercions.rst

```diff
@@ -186,9 +187,9 @@ Declaring Coercions
      This defines :token:`ident` just like :n:`Let @ident := @term  {? @type }`,
      and then declares :token:`ident` as a coercion between it source and its target.
 
-Assumptions can be declared as coercions at declaration time.
```

Comment:
Sorry for not noticing earlier, but this section was already in a pretty terrible shape and the loss of this introduction sentence really makes things harder to follow. I hope that you will come back to it when fixing the grammar of the corresponding commands. In the meantime could you replace the few paragraphs between the two command definitions by a more compact explanation like so:

```rst
Some objects can be declared as coercions at the point of their definition.
This is the case in particular of assumptions, constructors of inductive types
and record fields. To do so use :n:`:> @type` instead of :n:`: @type` to declare
their types. Cf. :n:`@of_type`.
```

(and remove the outdated `FIXME` at the same time)?

Reply:
Check what I wrote here.

Reply:
LGTM